frontend/src/locales/en-US.json

```diff
@@ -489,6 +489,11 @@
           "enable": "Disable self-service signup",
           "description": "Once disabled, users cannot self-service signup and can only be invited by the workspace owner."
         },
+        "require-2fa": {
+          "enable": "Require two-factor authentication for everyone",
```

Comment:
nit: wording "for everyone in the workspace"?

Reply:
How about "xxx for all users"